README.md

```diff
@@ -332,7 +332,7 @@ end
 | `instances`                | The fields used to fill values under the `instances` section in the integration configuration file.                                                                                                                                                                                            |
 | `init_config`              | The fields used to fill values under the the `init_config` section in the integration configuration file.                                                                                                                                                                                      |
 | `logs`                     | The fields used to fill values under the the `logs` section in the integration configuration file.                                                                                                                                                                                             |
-| `use_integration_template` | Set to `true` (recommended) to use the default template, which writes the values of `instances`, `init_config`, and `logs` in the YAML under their respective keys. This defaults to `false` for backward compatibility, but will default to `true` in a future major version of the cookbook. |
+| `use_integration_template` | Set to `true` (recommended) to use the default template, which writes the values of `instances`, `init_config`, and `logs` in the YAML under their respective keys. This defaults to `false` for backward compatibility, but defaults to `true` in a future major version of the cookbook. |
```

Comment:
I dunno, this instance is specifically talking about a future major version. "will default" is correct if we're going to keep that. Or maybe "might default"

Reply:
```suggestion
| `use_integration_template` | Set to `true` (recommended) to use the default template, which writes the values of `instances`, `init_config`, and `logs` in the YAML under their respective keys. This defaults to `false` for backward compatibility, but may default to `true` in a future major version of the cookbook. |
```